tap_counter/ViewController.swift

```diff
@@ -0,0 +1,55 @@
+//
+//  ViewController.swift
+//  tap_counter
+//
+//  Created by Michail on 9/10/16.
+//  Copyright © 2016 mishaparadising. All rights reserved.
+//
+
+import UIKit
+
+class ViewController: UIViewController {
+    
+    var counter = 0;
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        // Do any additional setup after loading the view, typically from a nib.
+        
+        tapCounter.text = String(counter);
```

Comment:
So regards naming conventions. I am pretty flexible on that but I'd like to see view's type in the name. THat makes code maintenance a lot easier. 

For example. In this case `tapCounter` might be anything - UIButton, UIView, UIImageView, etc. But if you will name it `tapCounterLabel` the type will be clear everywhere in the code and you don't need to go to the definition to see that. 

That rule should apply everywhere. 
`decreaseByOneButton` -> `decreaseByOneBarButton`, or better `undoBarButton`
`resetButton` -> `resetBarButton `